span_sinks.go

```diff
@@ -0,0 +1,350 @@
+package veneur
+
+import (
+	"container/ring"
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/Sirupsen/logrus"
+	lightstep "github.com/lightstep/lightstep-tracer-go"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/stripe/veneur/ssf"
+	"github.com/stripe/veneur/trace"
+)
+
+const datadogResourceKey = "resource"
+const datadogNameKey = "name"
+
+const lightStepOperationKey = "name"
+
+const totalSpansFlushedMetricKey = "worker.spans_flushed_total"
```

Comment:
Do you want to rebase this onto #228 when that build succeeds and then migrate to use the new Recorder?